Gemfile

```diff
@@ -68,6 +68,7 @@ group :development do
   gem 'binding_of_caller'
   gem "pg_query"
   gem 'prosopite'
+  gem 'yard'
```

Comment:
If we want to bring in this dependency, should we do that in a separate PR?

Reply:
Will do. I actually thought I'd already added it to codebase; I'll have to see...